compiler/src/parsing/well_formedness.re

```diff
@@ -33,7 +33,7 @@ let prepare_error =
           ~loc,
           "This character literal contains multiple characters: '%s'\nDid you mean to create the string \"%s\" instead?",
           cl,
-          cl,
+          Str.global_replace(Str.regexp({|"|}), {|\"|}, cl),
```

Comment:
Do we want to use [`String.escaped`](https://v2.ocaml.org/api/String.html) here? Maybe a question for @ospencer 

```suggestion
          String.escaped(cl),
```

Reply:
I don't think we want to use `escaped` here since the semantics are pretty different than Grain's. Whatever the user wrote (with escaped quotes) is fine.